src/test/GameWithLuaAccess.h

```diff
@@ -44,12 +44,7 @@ class GameWithLuaAccess : public Game
 {
 public:
     GlobalGameSettings ggs;
```

Comment:
Remove this. Passing it to Game() will copy it but one might wrongly assume this instance is still used. Just do `Game(GlobalGameSettings(), ...)` in Line 47

Reply:
This is actually causing your issue as it shadows Game::ggs. Remove and it works